h/resources.py

```diff
@@ -114,7 +115,12 @@ def __init__(self, request):
 
     def __getitem__(self, pubid):
         try:
-            return self.request.db.query(Organization).filter_by(pubid=pubid).one()
+            org = self.request.db.query(Organization).filter_by(pubid=pubid).one()
+
+            # Inherit global ACL. See comments in `AuthClientFactory`.
+            org.__parent__ = Root(self.request)
```

Comment:
Correct. Without this ACL inheritance, Pyramid will find the context object for the request (an `Organization`), look for a `__acl__` property on it, not find one, and then reject the view because the user "doesn't have the `admin_organizations`" permission.